src/services/health/healthService.ts

```
@@ -77,11 +76,26 @@ async function checkRabbitMQ(): Promise<HealthDetail> {
   }
 }
 
+async function checkStellarHorizon(): Promise<HealthDetail> {
+  try {
+    await withTimeout(
+      stellarClient.getServer().root(),
+      TIMEOUT_MS
+    );
```

ESLint

[error] Replace `⏎······stellarClient.getServer().root(),⏎······TIMEOUT_MS⏎····` with `stellarClient.getServer().root(),·TIMEOUT_MS`

(prettier/prettier)